tests/models/test_keypoint_detection.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+import unittest
+from pathlib import Path
+
+import responses
+from dotenv import load_dotenv
+from requests.exceptions import HTTPError
+
+from roboflow.models.keypoint_detection import KeypointDetectionModel
+from roboflow.util.prediction import PredictionGroup
+
+load_dotenv(Path("../../.env"))
+
+
+with open(Path("../annotations/keypoint-detection-annotations/MM2A_46_R_T_predictions.json")) as f:
```

Comment:
tests get run from root -- can you update:

```suggestion
with open(Path("tests/annotations/keypoint-detection-annotations/MM2A_46_R_T_predictions.json")) as f:
```